### PR TITLE
#1: Initialize project. Create manual http test

### PR DESCRIPTION
**Pull request by DMCHMB**
- test in folder ./http

- Add checkstyle gradle plugin

- Add Github Workflows for PR and Develop branch

### CODE REVIEW

**Review by sursindmitry — APPROVED** [review]

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    mavenCentral()
+    maven { url = uri("https://repo.spring.io/milestone") }
+    maven { url = uri("https://repo.spring.io/snapshot") }
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    runtimeOnly("org.postgresql:postgresql")
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
```
[the remark this answers is not in the document]
Comment (DMCHMB) [reply build.gradle.kts]: test comment

[the remark this answers is not in the document]
Comment (IlyaLisov) [reply build.gradle.kts]: test comment

**Review by IliaWizy — CHANGES_REQUESTED** [review]
подход 1

File: .github/workflows/workflowForDevelop.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
+name: Develop branch workflow
```
Comment (IliaWizy) [inline .github/workflows/workflowForDevelop.yml:1]: saas

File: .github/workflows/workflowForDevelop.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
```
Comment (IliaWizy) [inline .github/workflows/workflowForDevelop.yml:2]: asdsa

File: build.gradle.kts [hunk 4]
```diff
@@ -0,0 +1,56 @@
+plugins {
```
Comment (IliaWizy) [inline build.gradle.kts:1]: single cooment

**Review by IliaWizy — APPROVED** [review]

**Pull request merged**